### DISCUSSION TITLE
Xamarin Forms UWP "Access to the path 'C:\\xxx' is denied."

### DISCUSSION THREAD

**Discussion by GUOLDEV** [reactiveui/Akavache#897] [General]
I am getting an exception error "Access to the path 'C:\xxx' is denied." when trying to get data using "await BlobCache.UserAccount.GetAllObjects<T>();"
Is there a permission config that I am missing?

**Comment by glennawatson** [comment]
https://github.com/reactiveui/Akavache/blob/d851c4166e75434bb352a45323c1079ffe0e42f3/src/Akavache.Sqlite3/Registrations.cs#L56

Normally the path for the default directory is determined here.

So if the registrations is not running could be likely this.

I suspect maybe a target framework we don't support anymore due to legacy frameworks or something.

You can manually configure the instance yourself to point towards the appropriate folder.
